microsoft-edge/devtools-guide-chromium/whats-new/2022/12/devtools-108.md

```diff
@@ -0,0 +1,112 @@
+---
+title: What's new in DevTools (Microsoft Edge 108)
+description: "Microsoft Edge WebDriver is now available for macOS running on Apple M1. Improved DevTools navigation via Command Palette. Better support for high contrast mode in DevTools. In Focus Mode, disabling JavaScript now produces a warning icon for the Sources tool.  And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 12/01/2022
+---
+# What's New in DevTools (Microsoft Edge 108)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+<!-- find "todo" in this .md file -->
+
+
+<!-- ====================================================================== -->
+## Microsoft Edge WebDriver is now available for macOS running on Apple M1
+
+<!-- Subtitle: On Apple M1 devices, Microsoft Edge WebDriver was previously running under emulation. Now, it will run faster without emulation. -->
+
+As of Microsoft Edge 105, you can now download Microsoft Edge WebDriver for macOS running on Apple M1 devices.  Previously, Microsoft Edge WebDriver on macOS was running under emulation with [Rosetta](https://support.apple.com/HT211861).  With the **Mac M1** version, Microsoft Edge WebDriver has improved performance, because it no longer runs under emulation.
+
+To download Microsoft Edge WebDriver on Apple M1 devices:
+1. In Microsoft Edge, go to `edge://version`, and note the full version number.
+1. Go to [Microsoft Edge WebDriver](https://developer.microsoft.com/microsoft-edge/tools/webdriver/).
+1. Find the version of Microsoft Edge WebDriver that matches your specific version of Microsoft Edge, and then click the **Mac M1** download link:
+
+![WebDriver version numbers for download](./devtools-108-images/webdriver-versions.png)
+
+Thank you for your feedback on this issue!
+
+See also:
+* [Use WebDriver to automate Microsoft Edge](../../../../webdriver-chromium/index.md)
+* [Microsoft Edge WebDriver](https://developer.microsoft.com/microsoft-edge/tools/webdriver/)
+* [Issue #17: No support for Apple silicon?](https://github.com/MicrosoftEdge/EdgeWebDriver/issues/17)
+
+
+<!-- ====================================================================== -->
+## Improved DevTools navigation via Command Palette
+
+<!-- Subtitle: This update fixes a bug related to opening DevTools from the Command Palette in Microsoft Edge. -->
+
+The Microsoft Edge Command Palette is an experimental feature that allows users to perform common tasks in Microsoft Edge.  In previous versions of Microsoft Edge, some DevTools-related commands for opening subpanels (tabs) within a tool didn't navigate to the correct tool as expected.  This update fixes this bug, improving the usability of commands such as **Sources: Show Snippets**:
```

Comment:
```suggestion
The Microsoft Edge Command Palette is an experimental feature that allows users to perform common tasks in Microsoft Edge.  In previous versions of Microsoft Edge, DevTools-related commands for opening subpanel tabs within a tool didn't navigate to the correct tool as expected.  This update fixes this bug, improving the usability of commands such as **Sources: Show Snippets**:
```

Requesting this change because I think most readers will think of tabs as tools, whereas this change refers specifically to subpanels within a tool.